executor/logger/worker_test.go

```diff
@@ -224,3 +185,55 @@ func TestWorkerProducerLog(t *testing.T) {
 		}
 	}()
 }
+
+func TestWorkerProducerLogSSL(t *testing.T) {
+
+	config := tlsConfig(t)
+	p, err := kafka.NewProducer(&kafka.ConfigMap{
+		"bootstrap.servers":        "broker",
+		"debug":                    "all",
+		"socket.timeout.ms":        10,
+		"security.protocol":        "SSL",
+		"ssl.ca.location":          config.CaPEM,
+		"ssl.key.location":         config.KeyPEM,
+		"ssl.certificate.location": config.CertPEM,
+		"ssl.key.password":         config.KeyPassword})
+
+	// myOAuthConfig := "scope=myscope principal=gotest"
+	// p, err := kafka.NewProducer(&kafka.ConfigMap{
+	// 	"security.protocol":       "SASL_PLAINTEXT",
+	// 	"sasl.mechanisms":         "OAUTHBEARER",
+	// 	"sasl.oauthbearer.config": myOAuthConfig,
+	// })
+
```

Comment:
That part is only here if you wanna run a debug test and check what the producer is creating.